code/datums/abilities/flock/flockmind.dm

```diff
@@ -204,27 +204,35 @@
 
 /datum/targetable/flockmindAbility/healDrone
 	name = "Concentrated Repair Burst"
-	desc = "Fully heal a drone through acceleration of its repair processes."
+	desc = "Heal all flock units in an area."
```

Comment:
Could maybe leave in a message about how it's a partial heal, also leaving the message in for repair processes would be cool but not necessary